components/script/dom/element.rs

```diff
@@ -1132,18 +1142,23 @@ impl<'dom> LayoutElementHelpers<'dom> for LayoutDom<'dom, Element> {
         &(self.unsafe_get()).namespace
     }
 
+    fn get_lang_attr_val_for_layout(self) -> Option<&'dom str> {
+        if let Some(attr) = self.get_attr_val_for_layout(&ns!(xml), &local_name!("lang")) {
+            return Some(attr);
+        }
+        if let Some(attr) = self.get_attr_val_for_layout(&ns!(), &local_name!("lang")) {
+            return Some(attr);
+        }
+        None
+    }
```

Comment:
Can be left for a follow-up, but if JS changes the attribute, then this needs to be updated.